test/es6/unscopablesWithScopeTest.js

```diff
@@ -13,6 +14,24 @@ var tests = [
             assert.isTrue(Array.prototype.hasOwnProperty(Symbol.unscopables), "Array should have Array.prototype[@@unscopables] property");
         }
     },
+    {
+        name: "Check if all excepted properties exist in Array.prototype[@@unscopables] and have corresponding values",
+        body: function () 
+        {
+            
+            const unscopables = Array.prototype[Symbol.unscopables];
+
+            const list = ["copyWithin", "entries", "fill", "find", "findIndex", "flat", "flatMap", "includes", "keys", "values"];
+            const length = list.length;
+
+            for (let index = 0; index < length; index++)
+            {
+                const propName = list[index];
+                assert.areEqual(unscopables[propName], true, `Array.prototype[@@unscopables].${ propName } - \`true\` value excepted, got - \`${ value }\``);

```

Comment:
This failed as "value" is not defined - the `assert.areEqual` function is already going to print:
```
assert.areEqual failed
expected: true
actual: (whatever the value is) 
```
So can either get rid of the message or make it generic: `Array.prototype[@@unscopables].${ propName } should equal true`